### PR TITLE
glowing flower animation

### PR DESCRIPTION
**Pull request by saurabhsingh720**
fl
Glowing flower animation

### CODE REVIEW

**Comment by sophiabrandt** [comment]
@saurabhsingh720  Thanks for this contribution. 🎉 

Please check the following checklist that you need to fulfill for us to be able to merge your request.

Please see the [Guide in README.md](https://github.com/zero-to-mastery/Animation-Nation#how-to-contribute) for further details.

##  Checklist
- [x] created a submission into the `./Art` folder
- [ ] the contribution folder in `./Art` is the contributor's Github name
- [x] has a gif or png file of the animation
- [x] can find the html 
- [x] can find the style (css file)
- [ ] modified `include.js` **with the right details**

## Issues: 

- missing changes to `include.js`
- wrong folder name

Please refer to the `README.md` file for guidance and follow the guidelines.
If you have questions feel free to reach to the discord or ask them here.

**Comment by saurabhsingh720** [comment]
issue resolved please check it

File: include.js [hunk 1]
```diff
@@ -1682,6 +1682,14 @@ let cards = [
     author: 'Aditya',
     githubLink: 'https://github.com/adityaanandz'
   },
+
+  {
+    artName: 'flower-animation',
+    pageLink: './Art/flower-animation/index.html',
```
Comment (sophiabrandt) [inline include.js:1688]: This needs to correspond with the folder name, e.g. `Art/saurabhsingh720/index.html`

File: include.js [hunk 2]
```diff
@@ -1682,6 +1682,14 @@ let cards = [
...
+    imageLink: './Art/fl.gif',
```
Comment (sophiabrandt) [inline include.js:1689]: Same here.

**Review by sophiabrandt — CHANGES_REQUESTED** [review]

**Comment by saurabhsingh720** [comment]
@sophiabrandt  your requested change has been made. Please review it.

**Pull request merged**